### PR TITLE
make: set LIBBPFGO_OSRELEASE_FILE default value

### PR DESCRIPTION
**Pull request by geyslan**
### 1. Explain what the PR does

Fix: #3156

272672ee **make: set LIBBPFGO_OSRELEASE_FILE default value** _<sub>(2023/jun/12) \<Geyslan Gregório\></sub>_

```
This sets LIBBPFGO_OSRELEASE_FILE to "/etc/os-release-host" by default
if it is not set, so that the docker command can be run without
specifying the file.

It also updates the documentation to reflect this change.
```

### 2. Explain how to test it

`make -f builder/Makefile.tracee-container build-tracee`
`make -f builder/Makefile.tracee-container run-tracee`

### CODE REVIEW

**Review by rafaeldtinoco — APPROVED** [review]
LGTM

File: docs/docs/integrating/container-engines.md [hunk 1]
```diff
@@ -69,7 +68,6 @@ As a user, when container enrichment is enabled the event output will include en
         --pid=host --cgroupns=host --privileged \
         -v /etc/os-release:/etc/os-release-host:ro \
         -v /var/run/docker.sock:/var/run/docker.sock \
-        -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host \
         aquasec/tracee:{{ git.tag }} \
         trace --output json --containers
```
Comment (yanivagman) [inline docs/docs/integrating/container-engines.md:72]: Seems like we still have the trace subcommand in some places in the documentation, which don't support anymore

Comment (geyslan) [reply docs/docs/integrating/container-engines.md:72]: I'll raise other chore PR removing it.

Comment (geyslan) [reply docs/docs/integrating/container-engines.md:72]: https://github.com/aquasecurity/tracee/pull/3238

**Pull request merged**